### DISCUSSION TITLE
error F000: SearchString: Error while processing file

### DISCUSSION THREAD

**Discussion by zirobin** [tianocore/edk2#3408] [General]
Hi, I am porting a gui lib LVGL to UEFI, and use the stable version stable202111 to build it.
I had porting successfully by using the UDK2018, but it is too old.
When build the LVGL lib,it failed with an error: **error F000: SearchString: Error while processing file**
The file cause this error is an font array file, some font comment with an unicode char, I try to remove it and the file can build pass.
But there are so many fonts and files, anyway to solve this problem?

**Comment by Maxinator500** [comment]
I understand the conversation is very old, but still want to share how to fix the **error F000: SearchString: Error while processing file**. I encountered it while attempting to utilize STRING_TOKEN (UNI file) and IMAGE_TOKEN (IDF file) together in EDK 2025-02 stable release.

Editing a single line in _StrGather.py_ worked for me. The solution was to comment out the _raise_ command in _SearchString_ function.
